src/main/java/com/example/LoyaltyCard.java

```diff
@@ -0,0 +1,29 @@
+package com.example;
+
+import static com.google.common.base.Preconditions;
+
+
+/**
+ * Class implements Value Object.
+ */
+public class LoyaltyCard {
+    /**
+     * Customer's id.
+     */
+    private final String id;
+
+    /**
+     * Constructor for checking id with method checkNotNull.
```

Comment:
You don't need to duplicate what a method or a constructor does in Javadoc. You just document an intent. It should be something like:

```
Creates a new card with the passed number.
```